pinot-core/src/main/java/org/apache/pinot/core/query/executor/ServerQueryExecutorV1Impl.java

```diff
@@ -322,10 +320,7 @@ private InstanceResponseBlock executeInternal(ServerQueryRequest queryRequest, E
       _serverMetrics.addMeteredTableValue(tableNameWithType, ServerMeter.QUERY_EXECUTION_EXCEPTIONS, 1);
       instanceResponse = new InstanceResponseBlock();
       // Do not log verbose error for BadQueryRequestException and QueryCancelledException.
-      if (e instanceof BadQueryRequestException) {
-        LOGGER.info("Caught BadQueryRequestException while processing requestId: {}, {}", requestId, e.getMessage());
-        instanceResponse.addException(QueryException.getException(QueryException.QUERY_VALIDATION_ERROR, e));
-      } else if (e instanceof QueryCancelledException) {
+      if (e instanceof QueryCancelledException) {
```

Comment:
I asked in slack as well and not specific to this PR. Why is this pattern used instead `catch(QueryCancelledException) { ... } catch (Exception e) { ... }

Reply:
TBH, I don't know; probably for historical reasons or because everything is the same but the message being logged.